R/double_ml_plr.R

```diff
@@ -112,7 +127,7 @@ DoubleMLPLR = R6Class("DoubleMLPLR",
     #' A `character(1)` (`"partialling out"` or `IV-type`) or a `function()`
```

Comment:
Consistent format if `score` is a `character`

```suggestion
    #' A `character(1)` (`"partialling out"` or `"IV-type"`) or a `function()`
```

Reply:
Just a reminder: In case we implement this change, we'd have to render the documentation again

Reply:
Done